src/main/java/internet/market/controllers/AddProductToShoppingCartController.java

```diff
@@ -10,7 +10,7 @@
 import javax.servlet.http.HttpServletResponse;
 
 public class AddProductToShoppingCartController extends HttpServlet {
-    private static final Long USER_ID = 1L;
+    private static final String USER_ID = "user_Id";
```

Comment:
Just a remark) The task in HW said we need to 'remove all static variables `user_id`", so I'm not sure if we still need to keep the constant variable `USER_ID` or just use `user_id` as a parameter in `getAttribute()`. Both ways work

Reply:
@AnaLanda yes, both ways work)